### PR TITLE
[FIX] pos_loyalty: fix test

### PR DESCRIPTION
**Pull request by robinengels**
This commit fix unconsistent test.
https://runbot.odoo.com/web/#id=54900&view_type=form&model=runbot.build.error&menu_id=405&cids=1

opw-3583174
---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/pos_loyalty/static/src/tours/PosLoyaltyTour.js [hunk 1]
```diff
@@ -268,9 +268,12 @@ ProductScreen.do.clickPartnerButton();
 ProductScreen.do.clickCustomer('AAA Partner');
 ProductScreen.do.clickDisplayedProduct('Product B');
 ProductScreen.do.clickDisplayedProduct('Product A');
```
Comment (caburj) [inline addons/pos_loyalty/static/src/tours/PosLoyaltyTour.js:270]:
```suggestion
ProductScreen.check.totalAmountIs('100.00');
ProductScreen.do.clickDisplayedProduct('Product A');
```

Instead in saas-16.3, can we start at 16.0?

Comment (vlst-odoo) [reply addons/pos_loyalty/static/src/tours/PosLoyaltyTour.js:270]: but 16.3 is where the tests started crashing, no?

Comment (caburj) [reply addons/pos_loyalty/static/src/tours/PosLoyaltyTour.js:254]: No, there are also cases linked to it starting 16.0.

**Comment by caburj** [comment]
I rebased to make runbot green.

**Comment by caburj** [comment]
robodoo r+

**Pull request closed without merging**